src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3141,6 +3141,30 @@ public static void DCS_KnownTypeMethodName()
         Assert.Equal(value.emps[1].Name, actual.emps[1].Name);
     }
 
+    [Fact]
+    public static void DCS_NativeDll()
+    {
+        NativeDllWrapper.CallIntoNativeDll();
+    }
+
+    [Fact]
+    public static void DCS_ReadOnlyDictionaryCausingDuplicateInvalidDataContract()
```

Comment:
I move the DCJS part found that the rest of the code same with DCS_ReadOnlyDictionary, so I remove this test in DCS, just keep DCJS part test